distro/adaptation/debian-12

```diff
@@ -46,3 +46,14 @@ libpython3: libpython3.11
 gcc: gcc-12
 libglut3: libglut3.12
 libgsl: libgsl27
+python2:
+qemu: qemu-kvm
+qt5-default:
+mcelog: rasdaemon
+libldap-2.4-2: libldap-2.5-0
+cython: cython3
+dh-systemd: debhelper
+libevent-2.1-6: libevent-2.1-7
+libhwloc5: libhwloc15
+libssl1.1: libssl3
+python3-tox: tox
```

Comment:
there's no python3-tox in code directly, it only has python-tox. Pls simply rename python-tox to tox in the depends file.

Reply:
ok